bin/build-shared-version-info.js

```diff
@@ -51,7 +51,7 @@ async function gatherBuildInfo() {
   const packageObject = JSON.parse(await readFile(packageJSON, 'utf8'))
   return {
     VERSION: packageObject.version,
-    BUILD_TIMESTAMP: Date.now(),
+    BUILD_TIMESTAMP: process.env.SOURCE_DATE_EPOCH || Date.now(),
```

Comment:
I think this should work (I can test locally in a bit).
In https://reproducible-builds.org/docs/source-date-epoch/ they suggest the below (and I don't know the difference between `Date.now()` and `new Date().getTime()`, but from looking at your change though I am assuming that currently it would be  `String` when using the environment variable and an `Integer` when using the function call - is that intentional?).

```javascript
var timestamp = new Date(process.env.SOURCE_DATE_EPOCH ? (process.env.SOURCE_DATE_EPOCH * 1000) : new Date().getTime());

// Alternatively, to ensure a fixed timezone:

var now = new Date();
if (process.env.SOURCE_DATE_EPOCH) {
  now = new Date((process.env.SOURCE_DATE_EPOCH * 1000) + (now.getTimezoneOffset() * 60000));
}
```

Reply:
interesting to know that timestamp is in seconds instead of milliseconds. anyways does not matter much, because currently this is just used for logging it at app startup